src/customFileParser/unpackGeometry/Basic.ts

```diff
@@ -1,36 +1,37 @@
-import { RenderedGeometryGroups } from '../sharedFileParserTypes';
 import PropertyLoader from '../PropertyLoader';
-import { zAxis } from '../../constants';
+import { xAxis, zAxis } from '../../constants';
 import * as THREE from 'three';
 
 const centerA = new THREE.Vector3();
 const centerB = new THREE.Vector3();
+const localXAxis = new THREE.Vector3();
+const axisRotation = new THREE.Quaternion();
 
-function addBox(groups: RenderedGeometryGroups, data: PropertyLoader) {
-  groups.Box.add(data.nodeId, data.treeIndex, data.color, data.center, data.normal,
+function addBox(groups: {[name: string]: any}, data: PropertyLoader) {
```

Comment:
Why `any` type here? Seems like it should be GeometryGroup or something?

Reply:
Fixed

Reply:
(Everywhere)